### PR TITLE
Issues 9494 9645

### PR DESCRIPTION
**Pull request by lbai001**
#### Proposed changes:
Discharge endDate limit -> 730 when env is not production
Discharge endDate limit is still 180 when env is production
Removed vets_application_status_slow call from ApplicationStatus.jsx
#### Files changed: 
        modified:   src/js/common/schemaform/save-in-progress/ApplicationStatus.jsx
        modified:   src/js/hca/validation.js
        modified:   test/hca/validation.unit.spec.js
#### Screenshots
screen shot 2018-04-04 at 2 52 31 pm
screen shot 2018-04-04 at 2 53 07 pm

### CODE REVIEW

File: src/js/hca/validation.js [hunk 1]
```diff
@@ -3,14 +3,27 @@ import _ from 'lodash/fp';
 import { convertToDateField, validateCurrentOrPastDate } from '../common/schemaform/validation';
 import { isValidDateRange } from '../common/utils/validations';
 
+function calculateEndDate() {
+  let endDateLimit;
+  if (process.env.BUILDTYPE === 'production') {
```
Comment (jbalboni) [inline src/js/hca/validation.js]: We normally use `__BUILDTYPE__` instead of `process.env.BUILDTYPE`

**Review by jbalboni — COMMENTED** [review]
One small comment about the build type flag, but otherwise the code looks good. I want to have Peggy approve the message before merging this.

File: src/js/hca/validation.js (hunk 1 above)
Comment (lbai001) [reply src/js/hca/validation.js]: ok

**Comment by rtluu** [comment]
@kellyskennedy @DanielleSoCompany, @peggygannon is OOO until next week and our team was wondering if either of you could give us some quick feedback regarding how we should describe 730 days to a veterans. We're thinking '2 years' or '24 months'. What do you think?

**Comment by DanielleSoCompany** [comment]
Hi @rtluu sure, happy to help! We usually go with 2 years. Thanks!

**Review by jbalboni — APPROVED** [review]

**Pull request merged**